site/content/en/docs/release/_index.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
move this under contribution

Reply:
But if someone has no interest in contributing to the project, they're just looking for the release schedule I don't think they'd look for it under `Contribution`.

Reply:
But I guess release schedule only really matters to contributers.

Reply:
Done